internal/cmd/project/create/create.go

```diff
@@ -82,8 +108,8 @@ func createProject(f *cmdutil.Factory, projectName string) error {
 }
 
 func paramCheck(opts *Options) error {
-	if opts.ProjectName == "" {
-		return fmt.Errorf("please specify project name with --name")
+	if opts.ProjectRegion == "" {
+		return fmt.Errorf("please specify project region with --region")
 	}
```

Comment:
I think `opts.ProjectName` should still be checked 🤔

Reply:
It's optional now. Do u mean that we should check if it's legal?